CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+
+1. Find the module you are interested in, test it and if you find any issues then feel free to open pull requests.
+2. Documentation is an integral part of our repository. Submit updates and improvements to documentations.
+3. Help others by answering on Stack overflow, Github or any other technical forum
+4. Share with us any new exciting feature you might have in mind 
+
+## How to Contribute
+
+### Reporting Bugs
+
+1. Update to the most recent main release. This will help you to be in sync with any recent fixes we have made or merged.
+2. Search for similar issues because it's highly possible that someone has encountered it already and started working on it.
+3. For any bug fix, we expect contributors to create a sandbox environment indicating the bug. This will help us to verify the bug. If you can't provide a working sandbox environment, then we will request you to provide us detailed steps on how to reproduce the bug.
+4. Once you have raised the issue, one of the core member will get back to you and will provide feedback.
+5. Please make sure to be active on your raised issue. Your issue might be closed if it becomes stale.
+6. If possible, submit a Pull Request with a failing test. Best if you can also fix the issue in the same PR.
+
+We review issues and Pull Requests frequently and we will try to reach back as soon as possible.
+
+### Open Pull Request
+
+Once your issue has been reviewed and accepted, then you can also create a Pull Request and own the issue. Simply go to Pull Request tab and create a PR.
+
+### Submit PR and get feedback
+
+1. After submitting your PR, the first thing you would want to do will be a self review. Barring special circumstances, maintainers will not review PRs until all checks pass (Travis, AppVeyor, etc).
+2. One or more maintainers will use GitHub's review feature to review your PR.
+3. If the maintainer asks for any changes, edit your changes, push, and ask for another review. Additional tags (such as `needs-tests`) will be added depending on the review.
+4. If the maintainer decides to pass on your PR, they will thank you for the contribution and explain why they won't be accepting the changes. That's ok! We still really appreciate you taking the time to do it, and we don't take that lightly.
+5. If your PR gets accepted, it will be marked as such, and you'll be able to request the core team to merge.
+
+### Hard work paid off
+
+Congratulations on succesfully contributing and the whole community thanks you!!!!
+
+Once you PR is merged, you will be added to the contributors list.
+
+
+## Need more help?
+
+If for any reason you have questions on the process, feel free to reach us back at [igupta@ncsu.edu](mailto:ktummur@ncsu.edu). Github process can be very complex and we dont want to loose your valuable contributions because of that. We are very excited that you have visited us and you will make our project much better.
```

Comment:
in [], it has my email id and in () it has yours. Can you proof read this once?

Reply:
Changed it, thank you!